srai/loaders/geoparquet_loader.py

```diff
@@ -0,0 +1,68 @@
+"""
+Geoparquet loader.
+
+This module contains geoparquet loader implementation.
+
+"""
+
+from pathlib import Path
+from typing import List, Optional, Union
+
+import geopandas as gpd
+
+
+class GeoparquetLoader:
+    """
+    GeoparquetLoader.
+
+    Geoparquet [1] loader is a wrapper for a `geopandas.read_parquet` function
+    and allows for an automatic index setting and additional geometry clipping.
+
+    References:
+        [1] https://github.com/opengeospatial/geoparquet
+
+    """
+
+    def load(
+        self,
+        file_path: Union[Path, str],
+        index_column: Optional[str] = None,
+        columns: Optional[List[str]] = None,
+        area: Optional[gpd.GeoDataFrame] = None,
+    ) -> gpd.GeoDataFrame:
+        """
+        Loads a geoparquet file.
```

Comment:
use imperative mode -> "Loads a geoparquet file."

Reply:
Changed to `Load a geoparquet file.`

Reply:
Great! Sorry for misspelling